event-bridge/src/main/java/CrimeBridge.java

```diff
@@ -1,3 +1,5 @@
+// camel-k: language=java
```

Comment:
Remove also maven directory structure (`/src/main/java..`), please.

Reply:
Let's keep this for last and review it a later time. There's more code coming to this demo and I'd like to keep things organized.